Makefile

```diff
@@ -63,9 +66,19 @@ test_browser: ## run browser tests
 	yarn wait-port localhost:9000
 	yarn test:browser-headless
 
-test_browser_containerized:
-	docker build . -f browsertests.Dockerfile --tag libero/reviewer-browsertests:${IMAGE_TAG}
-	docker run --network reviewer -e BASE_URL="reviewer-client_nginx:9000" libero/reviewer-browsertests:${IMAGE_TAG}
+test_chromium:
+	docker run --network reviewer \
+		-e BASE_URL="reviewer-client_nginx:9000" \
+		libero/reviewer-browsertests:${IMAGE_TAG}
+
+test_firefox:
+	docker run --network reviewer \
+		-e BASE_URL="reviewer-client_nginx:9000" \
+		--entrypoint testcafe \
+		libero/reviewer-browsertests:${IMAGE_TAG} \
+		"firefox:headless" 'tests/**/*.browser.ts'
+
+test_browser_containerized: build_browsertest test_chromium test_firefox
```

Comment:
Does this get used anywhere? I noticed you run these commands separately in the Action.